sentences/sv/_common.yaml

```diff
@@ -54,12 +54,15 @@ expansion_rules:
   brightness: "{brightness} [percent]"
   slå_på: "( tänd | sätt på | slå på | starta )"
   slå_av: "( släck | stäng av | stoppa )"
-  i_på: "(i | på)"
+  i_på: "(i | på | vid | bredvid)"
   temp: "(temperatur)"
-  alla: "(all | alla )"
+  alla: "(all | alla | allt)"
+  ljusintensitet: "(ljusstyrka | intensitet | styrka | nivå | effekt)"
   lampa: "(ljuset | belysning[en] | lampan | lampor[na])"
+  ljuskällor: "(ljuskällor[na] | lampor[na] | belysning[en] | ljus[en] | glödlampor[na] | lysrör[en])"
```

Comment:
Should probably merge "lampa" with "ljuskällor" to use same in "TurnOn/TurnOff" as in "Set"

Reply:
Either that or have one ”ljuskälla” and one ”ljuskällor” to give the possibility to distinguish between singular and plural forms?